application/controllers/endpoint_controller.php

```diff
@@ -0,0 +1,190 @@
+<?php if (!defined('BASEPATH')) {
+    exit('No direct script access allowed');
+}
+
+require APPPATH . '/libraries/REST_Controller.php';
+
+class Endpoint extends REST_Controller
+{
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->load->model('eventos_model');
+        //$this->load->library('form_validation');
+    }
+
+    /*
+    |
+    |    SELECT
+    |
+     */
+    public function method_get()
+    {
+        $result = $this->the_model->function();
+
+        if ($result['code'] == 0) {
+            $this- > response([
+                'code'   => 2000,
+                'title'  => 'FILA',
+                'detail' => $result['row'],
+            ], $this::HTTP_OK);
+        } else {
+            $this- > response(
+                [
+                    'code'   => 2004,
+                    'title'  => 'CONSULTA VACÍA',

```

Comment:
Could you please update to use the appropriate language files (same goes for everything)?